driver-core/src/main/java/com/datastax/driver/core/Metrics.java

```diff
@@ -273,6 +273,7 @@ void shutdown() {
         private final Counter writeTimeouts = registry.counter("write-timeouts");
```

Comment:
Also, adding a new method to a class does not break binary compatibility as long as the client doesn't need to implement it (e.g. if it's abstract). See [here](https://wiki.eclipse.org/Evolving_Java-based_APIs_2).